plugins/org.jboss.tools.openshift.core/src/org/jboss/tools/openshift/internal/core/models/PortSpecAdapter.java

```diff
@@ -99,9 +99,15 @@ public boolean equals(Object obj) {
 		return true;
 	}
 	
-	
-	
-	private String getName(int port, String protocol) {
+    /**
+     * For better test reporting
+     */
+    @Override
+    public String toString() {
+        return "PortSpecAdapter [name=" + name + ", protocol=" + protocol + ", port=" + port + "]";
```

Comment:
Yes StringBuilder is an anti pattern for toString method see this presentation (in French but Youtube translation may help): https://www.youtube.com/watch?v=U97zT8kA7G4